WPFHospitalEditor/Controller/Interface/IRenovationServerController.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WPFHospitalEditor.Controller.Interface
+{
+    public interface IRenovationServerController
+    {
+        string ScheduleRenovation(DateTime startDate, DateTime endDate, int doctorId, int patientId);
```

Comment:
you should use TimeInterval class for start and end date

Reply:
Done